### PR TITLE
add sub-pages to D&A

### PR DESCRIPTION
**Pull request by pochedls**
This PR is to add sub-pages to the Detection and Attribution Group. We've also added publications dating back to 2012.

### CODE REVIEW

File: research/DandA/data.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+layout: detect
```
Comment (mauzey1) [inline research/DandA/data.md:2]:
```suggestion
layout: default
```

File: research/DandA/index.md [hunk 2]
```diff
@@ -1,65 +1,10 @@
+--- 
+layout: detect
```
Comment (mauzey1) [inline research/DandA/index.md:2]:
```suggestion
layout: default 
```

File: research/DandA/people.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+--- 
+layout: detect
```
Comment (mauzey1) [inline research/DandA/people.md:2]:
```suggestion
layout: default 
```

File: research/DandA/publications.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
+--- 
+layout: detect
```
Comment (mauzey1) [inline research/DandA/publications.md:2]:
```suggestion
layout: default 
```

File: research/DandA/index.md (hunk 2 above)
Comment (mauzey1) [reply research/DandA/index.md:2]: There are no links on this page to the data, people, and publications pages.

File: research/DandA/data.md (hunk 1 above)
Comment (pochedls) [reply research/DandA/data.md:2]: @mauzey1 - part of the goal of this commit was to add the links (as in the Cloud Feedbacks Project). Can I keep this set as `detect.md`?

Same for `index.md`, `people.md`, and `publications.md`

Comment (mauzey1) [reply research/DandA/data.md:2]: There is currently no layout called detect in the _layout directory.  Will that be part of this commit?

Comment (pochedls) [reply research/DandA/data.md:2]: Sorry @mauzey1 - I thought it was in the commit. It is added now.

**Pull request merged**